Documentation/teaching/info/vm.rst

```diff
@@ -4,118 +4,118 @@
 Virtual Machine Setup
 =====================
 
-Exercises are designed to run on a qemu based virtual machine. In
-order to run the virtual machine you will need following packages:
+Practice work is to designed to run on a QEMU based virtual machine. In order
+to run and use the virtual machine the following packages are required:
 
-* build-essential
-* qemu-system-x86
-* qemu-system-arm
-* kvm
-* python3
+* ``build-essential``
+* ``qemu-system-x86``
+* ``qemu-system-arm``
+* ``kvm``
+* ``python3``
+* ``minicom``
 
 The virtual machine setup uses prebuild Yocto images that it downloads
-from downloads.yocyoproject.org and a kernel image that it builds
+from downloads.yoctoproject.org and a kernel image that it builds
```

Comment:
Maybe make downloads.yoctoproject.org an url.

Reply:
I'll remove the hostname / URL as it's part of the `YOCTO_URL` variable in `tools/labs/qemu/Makefile`. In case we change it, we won't have to change it here.